src/Views/IframeContentView.php

```diff
@@ -45,6 +45,24 @@ class IframeContentView
      */
     protected $bodyClasses = ['give-form-templates'];
 
+    /**
+     * Get the Form ID based on the body content.
+     *
+     * @return int|null
+     */
+    protected function getFormIdFromBody()
+    {
+        $form_id = null;
+
+        $prefix = '<input type="hidden" name="give-form-id" value';
+        $re     = '/' . preg_quote($prefix) . '=([\'"])?((?(1).+?|[^\s>]+))(?(1)\1)/is';
+        if (preg_match($re, $this->body, $match)) {
+            $form_id = absint($match[2]);
+        }
+
+        return $form_id;
+    }
+
```

Comment:
@glaubersilva while this is clever, this view class is very much declarative so this approach doesn't quite fit. Instead, I think we're better off creating a `setFormId()` getter & `$formId` property, then travel up to the instances where the `$iframeView = new IframeContentView();` is created and `setFormId()` there.  

Notice in `src/Views/Form/defaultFormTemplate.php` there is already a way to get the formId before creating the view instance. `$formId = FrontendFormTemplateUtils::getFormId();`

Back in the render method you would then just have:
```
echo apply_filters('the_title', $this->title, $this->formId); ?></title>
```

Reply:
@jonwaldstein Thanks! I got your point and already did the changes.

Reply:
Awesome, thanks!